index.html

```diff
@@ -8,14 +8,32 @@
 
   <body>
     <h1>Tic Tac Toe</h1>
-    <div id="tic-tac-toe"></div>
+    <div id="tic-tac-toe">
+      <table class="board">
+        <tr class = "row">
+          <td class="cell 0" ></td>
```

Comment:
There's a cool feature in HTML where you don't have to put extra data into the class attribute. The numbers don't really belong here because they aren't about formatting. Instead, you can add a data-* attribute where you store the extra stuff. For example, here you could do `<td class="cell" data-cell-number=0 ></td>`